_data/education.yml

```diff
@@ -34,6 +34,16 @@ websites:
       facebook: babbel.languages
       email_address: support@en.babbel.com
 
+    - name: California Institute of Technology
+      url: https://www.caltech.edu
+      img: caltech.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      exceptions:
+          text: "sms is required to activate"
```

Comment:
On similar sites we've had the exception text `SMS-capable phone required for initial setup.`. Would that work here?

Reply:
@Sunit-Kulkarni last thing to change ^

Reply:
Yup that should work there.

Reply:
just updated it